### PR TITLE
#502 Again - Makes it easier for ops to op them selves.

### PR DESCRIPTION
**Pull request by AlexFreedomMod**
This is life #502. But i just changed it to make it work.

### CODE REVIEW

**Comment by AlexFreedomMod** [comment]
Testing.

**Comment by Wild1145** [comment]
CURLY BRASSES, FOR THE LOVE OF GOD USE CURLY BRASSES.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
-@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Automatically ops user.", usage = "/<command>")
+@CommandPermissions(level = AdminLevel.ALL, source = SourceType.ONLY_IN_GAME)
+@CommandParameters(description = "Automatically ops the sender.", usage = "/<command>")
 public class Command_opme extends TFM_Command
 {
     @Override
     public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
     {
+         if (!TFM_AdminList.isAdminImpostor(sender_p))
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Please look at this, you dont even need to test this to know that there is no way that will compile...

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
         TFM_Util.adminAction(sender.getName(), "Opping " + sender.getName(), false);
         sender.setOp(true);
         sender.sendMessage(TFM_Command.YOU_ARE_OP);
+         if (TFM_AdminList.isAdminImpostor(sender_p))
```
Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: Same here, this simply would not compile, let alone run.

Comment (Wild1145) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java]: This should actually be the else of the if else statement...

**Comment by AlexFreedomMod** [comment]
@Wild1145 Guess what. It compiled

**Comment by Wild1145** [comment]
@AlexFreedomMod Then you have disabled every and all checks to stop the compiling, because that should not compile, and will not work for as long as you have a hold in your ass.

**Comment by AlexFreedomMod** [comment]
I'll fix  it

**Pull request closed without merging**